src/neural_de.jl

```diff
@@ -179,13 +179,13 @@ function (n::NeuralDSDE{M})(x,p=n.p) where {M<:FastChain}
     solve(prob,n.args...;sensealg=TrackerAdjoint(),n.kwargs...)
 end
 
-function (n::NeuralDSDE{M})(x,p1,p2,st1,st2) where {M<:Lux.AbstractExplicitLayer}
+function (n::NeuralDSDE{M})(x,p,st1,st2) where {M<:Lux.AbstractExplicitLayer}
```

Comment:
julia can unpack it
```suggestion
function (n::NeuralDSDE{M})(x,(p1, p2),st1,st2) where {M<:Lux.AbstractExplicitLayer}
```

Reply:
Actually we need them to be packed in one p so that we can pass it to the SDEProblem 
https://github.com/Abhishek-1Bhatt/DiffEqFlux.jl/blob/b3b7ae7208f03918bdaaf67b39b6775515dcc6e0/src/neural_de.jl#L193

which will then call our functions where we will dereference individual p1 and p2 from p

Reply:
That's why I added parentheses it works with one `p` and Julia will unpack it like
```julia
p1, p2 = p
```

Reply:
Oh, sorry. Now, I understand!